cmd/main.go

```diff
@@ -157,8 +159,13 @@ func checkMainSyntax(c *cli.Context) {
 func Main() {
 	app := registerApp()
 	app.Before = func(c *cli.Context) error {
+
+		configDir := c.GlobalString("config-dir")
+		if configDir == "" {
+			fatalIf(errors.New("config-dir is empty"), "Unable to get config file.")
```

Comment:
Rename this as `fatalf(errors.New("Config directory is empty"), "Unable to get config file.")`